config/routes.rb

```diff
@@ -12,4 +12,12 @@
   end
 
   resources :product_questions, only: [:create]
+
+  namespace :api, defaults: { format: 'json' } do
+    namespace :v2 do
+      namespace :storefront do
+        resources :product_questions, only: [:index]
```

Comment:
@panuza did you test this api?
should have been nested inside product to work,
eg: `products/:product_id/product_questions`

Reply:
Thanks for updating routes.